### PR TITLE
CHSDI update ch.bafu.wasser-vorfluter

### PR DESCRIPTION
**Pull request by Simon1193**
[testlink](https://mf-chsdi3.dev.bgdi.ch/ltbei_wasser_vorfluter/shorten/86ed238715)

### CODE REVIEW

**Comment by ltclm** [comment]
@procrastinatio  do we have to be concerned if some of the codebuild tests are not working?

**Comment by procrastinatio** [comment]
The codebuild for python2 is missing some files in your branch. I did update it, should work now.

**Comment by procrastinatio** [comment]
But for now we don't have any integration tests on codebuild.

File: chsdi/models/vector/bafu.py [hunk 1]
```diff
@@ -1098,19 +1098,23 @@ class Gewaesserschutzbereiche (Base, Vector):
...
-    __tablename__ = 'vorfluter'
+    __tablename__ = 'effluent'
     __table_args__ = ({'schema': 'wasser', 'autoload': False})
     __bodId__ = 'ch.bafu.wasser-vorfluter'
     __template__ = 'templates/htmlpopup/vorfluter.mako'
-    __label__ = 'teilezgnr'
+    __label__ = 'id'
     id = Column('bgdi_id', Integer, primary_key=True)
-    teilezgnr = Column('teilezgnr', Integer)
+    id_2 = Column('id', Integer)
```
Comment (ltclm) [inline chsdi/models/vector/bafu.py]: why do you need to expose both ids bgdi_id and id (from the data , which is primary key). i would only expose the id which comes with the data

Comment (Simon1193) [reply chsdi/models/vector/bafu.py]: ID is an attribute from the table delivered by bafu and must apparear in the tooltip as specified in the data integration form.

However as they didn't specifiy a stabile id in the data integration form to be used, I created one with the table (the bgdi_id)

Comment (Simon1193) [reply chsdi/models/vector/bafu.py]: Ok I will use these id. thanks for the feedback

**Pull request merged**